### PR TITLE
[GraphBolt][CUDA] CSRToCOO for NeighborSampling

### PR DESCRIPTION
**Pull request by mfbalin**
## Description
This operator is required for efficient implementation of NeighborSampling in #6770.

### CODE REVIEW

File: graphbolt/src/cuda/csr_to_coo.cu [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+                    output_buffer + i, buffer_sizes + i,
+                    std::min(num_rows - i, max_copy_at_once), stream));
+              }
+            }));
+        return csr_rows;
+      }));
+}
+
+}  // namespace ops
+}  // namespace graphbolt
```
Comment (frozenbugs) [inline graphbolt/src/cuda/csr_to_coo.cu:92]: nit: newline.

**Review by frozenbugs — APPROVED** [review]
Do you plan to add unit test for this ops soon?

**Comment by mfbalin** [comment]
@Rhett-Ying the example test fails very often. I think the threshold should be lowered.
```
=========================== short test summary info ============================
FAILED tests/examples/test_sparse_examples.py::test_hgnn - AssertionError: assert 0.65 &gt; 0.66
 +  where 0.65 = float('.650\n')
=================== 1 failed, 11 passed in 124.52s (0:02:04) ===================
```

**Comment by mfbalin** [comment]
@frozenbugs is there a way to ignore the example CI failure and merge it anyway?

**Pull request merged**